ULogViewer/ViewModels/Session.cs

```diff
@@ -1346,6 +1556,22 @@ void UpdateDisplayableLogComparison()
 		}
 
 
+		// Update comparison for marked log info.
+		void UpdateMarkedLogInfoComparison()
```

Comment:
Seems no need to sort? ```MarkedLogInfo``` with same ```FileName``` whill be placed continuously because you add them by ```AddRange()```.